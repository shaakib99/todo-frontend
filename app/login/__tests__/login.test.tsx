import renderer from "react-test-renderer";
import Login from "../page";

describe("<Login />", () => {
  it("Match Snapshot", () => {
    const tree = renderer.create(<Login />);
    expect(tree).toMatchSnapshot();
  });
});
