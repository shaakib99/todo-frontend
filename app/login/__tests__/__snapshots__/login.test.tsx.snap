// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login /> Match Snapshot 1`] = `
<section
  className="h-auto min-w-min  m-5 w-11/12 md:w-1/2 py-5 px-2 flex flex-col items-center shadow-md"
  style={
    {
      "maxWidth": "1000px",
    }
  }
>
  <h1
    className="font-bold text-3xl m-5"
  >
    Login
  </h1>
  <form
    className="ant-form ant-form-horizontal css-dev-only-do-not-override-15rg2km flex flex-col items-center justify-center w-full px-24"
    onReset={[Function]}
    onSubmit={[Function]}
  >
    <div
      className="ant-form-item w-11/12 m-2 css-dev-only-do-not-override-15rg2km"
    >
      <div
        className="ant-row ant-form-item-row css-dev-only-do-not-override-15rg2km"
        style={{}}
      >
        <div
          className="ant-col ant-col-6 ant-form-item-label css-dev-only-do-not-override-15rg2km"
          style={{}}
        >
          <label
            className="ant-form-item-required ant-form-item-no-colon"
            htmlFor="email"
            title="Email"
          >
            Email
          </label>
        </div>
        <div
          className="ant-col ant-col-18 ant-form-item-control css-dev-only-do-not-override-15rg2km"
          style={{}}
        >
          <div
            className="ant-form-item-control-input"
          >
            <div
              className="ant-form-item-control-input-content"
            >
              <input
                aria-required="true"
                className="ant-input css-dev-only-do-not-override-15rg2km h-10"
                disabled={false}
                id="email"
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                style={{}}
                type="text"
                value=""
              />
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="ant-form-item w-11/12 m-2 css-dev-only-do-not-override-15rg2km"
    >
      <div
        className="ant-row ant-form-item-row css-dev-only-do-not-override-15rg2km"
        style={{}}
      >
        <div
          className="ant-col ant-col-6 ant-form-item-label css-dev-only-do-not-override-15rg2km"
          style={{}}
        >
          <label
            className="ant-form-item-required ant-form-item-no-colon"
            htmlFor="password"
            title="Password"
          >
            Password
          </label>
        </div>
        <div
          className="ant-col ant-col-18 ant-form-item-control css-dev-only-do-not-override-15rg2km"
          style={{}}
        >
          <div
            className="ant-form-item-control-input"
          >
            <div
              className="ant-form-item-control-input-content"
            >
              <span
                className="ant-input-affix-wrapper ant-input-password h-10 css-dev-only-do-not-override-15rg2km"
                onClick={[Function]}
              >
                <input
                  aria-required="true"
                  className="ant-input css-dev-only-do-not-override-15rg2km"
                  disabled={false}
                  hidden={null}
                  id="password"
                  onBlur={[Function]}
                  onChange={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  type="password"
                  value=""
                />
                <span
                  className="ant-input-suffix"
                >
                  <span
                    aria-label="eye-invisible"
                    className="anticon anticon-eye-invisible ant-input-password-icon"
                    onClick={[Function]}
                    onMouseDown={[Function]}
                    onMouseUp={[Function]}
                    role="img"
                    tabIndex={-1}
                  >
                    <svg
                      aria-hidden="true"
                      data-icon="eye-invisible"
                      fill="currentColor"
                      focusable="false"
                      height="1em"
                      viewBox="64 64 896 896"
                      width="1em"
                    >
                      <path
                        d="M942.2 486.2Q889.47 375.11 816.7 305l-50.88 50.88C807.31 395.53 843.45 447.4 874.7 512 791.5 684.2 673.4 766 512 766q-72.67 0-133.87-22.38L323 798.75Q408 838 512 838q288.3 0 430.2-300.3a60.29 60.29 0 000-51.5zm-63.57-320.64L836 122.88a8 8 0 00-11.32 0L715.31 232.2Q624.86 186 512 186q-288.3 0-430.2 300.3a60.3 60.3 0 000 51.5q56.69 119.4 136.5 191.41L112.48 835a8 8 0 000 11.31L155.17 889a8 8 0 0011.31 0l712.15-712.12a8 8 0 000-11.32zM149.3 512C232.6 339.8 350.7 258 512 258c54.54 0 104.13 9.36 149.12 28.39l-70.3 70.3a176 176 0 00-238.13 238.13l-83.42 83.42C223.1 637.49 183.3 582.28 149.3 512zm246.7 0a112.11 112.11 0 01146.2-106.69L401.31 546.2A112 112 0 01396 512z"
                      />
                      <path
                        d="M508 624c-3.46 0-6.87-.16-10.25-.47l-52.82 52.82a176.09 176.09 0 00227.42-227.42l-52.82 52.82c.31 3.38.47 6.79.47 10.25a111.94 111.94 0 01-112 112z"
                      />
                    </svg>
                  </span>
                </span>
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="ant-form-item w-11/12 m-2 css-dev-only-do-not-override-15rg2km"
    >
      <div
        className="ant-row ant-form-item-row css-dev-only-do-not-override-15rg2km"
        style={{}}
      >
        <div
          className="ant-col ant-col-6 ant-form-item-label css-dev-only-do-not-override-15rg2km"
          style={{}}
        >
          <label
            className="ant-form-item-no-colon"
            htmlFor="rememberme"
            title=" "
          >
             
          </label>
        </div>
        <div
          className="ant-col ant-col-18 ant-form-item-control css-dev-only-do-not-override-15rg2km"
          style={{}}
        >
          <div
            className="ant-form-item-control-input"
          >
            <div
              className="ant-form-item-control-input-content"
            >
              <label
                className="ant-checkbox-wrapper ant-checkbox-wrapper-in-form-item css-dev-only-do-not-override-15rg2km"
              >
                <span
                  className="ant-checkbox css-dev-only-do-not-override-15rg2km"
                >
                  <input
                    checked={false}
                    className="ant-checkbox-input"
                    disabled={false}
                    id="rememberme"
                    onChange={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="ant-checkbox-inner"
                  />
                </span>
                <span>
                  Remember me
                </span>
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="ant-form-item w-11/12 css-dev-only-do-not-override-15rg2km"
    >
      <div
        className="ant-row ant-form-item-row css-dev-only-do-not-override-15rg2km"
        style={{}}
      >
        <div
          className="ant-col ant-col-6 ant-form-item-label css-dev-only-do-not-override-15rg2km"
          style={{}}
        >
          <label
            className="ant-form-item-no-colon"
            title=" "
          >
             
          </label>
        </div>
        <div
          className="ant-col ant-col-18 ant-form-item-control css-dev-only-do-not-override-15rg2km"
          style={{}}
        >
          <div
            className="ant-form-item-control-input"
          >
            <div
              className="ant-form-item-control-input-content"
            >
              <button
                className="ant-btn css-dev-only-do-not-override-15rg2km ant-btn-default bg-primary outline-none border-none text-md px-5 py-2 h-auto font-light"
                disabled={false}
                onClick={[Function]}
                type="submit"
              >
                <span>
                  Login
                </span>
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </form>
  <div
    className="ant-divider css-dev-only-do-not-override-15rg2km ant-divider-horizontal ant-divider-with-text ant-divider-with-text-center"
    role="separator"
  >
    <span
      className="ant-divider-inner-text"
      style={{}}
    >
      Or, Login with 
    </span>
  </div>
  <section
    className="flex items-center justify-center w-full gap-2"
  >
    <div
      className="p-2 rounded shadow-md cursor-pointer"
    >
      <img
        alt="Google Logo"
        data-nimg="1"
        decoding="async"
        height={40}
        loading="lazy"
        onError={[Function]}
        onLoad={[Function]}
        src="/_next/image?url=%2Fassets%2Fimages%2Fgoogle-logo.png&w=96&q=75"
        srcSet="/_next/image?url=%2Fassets%2Fimages%2Fgoogle-logo.png&w=48&q=75 1x, /_next/image?url=%2Fassets%2Fimages%2Fgoogle-logo.png&w=96&q=75 2x"
        style={
          {
            "color": "transparent",
          }
        }
        title="Login with Google"
        width={40}
      />
    </div>
    <div
      className="p-2 rounded shadow-md cursor-pointer"
    >
      <img
        alt="Facebook Logo"
        data-nimg="1"
        decoding="async"
        height={40}
        loading="lazy"
        onError={[Function]}
        onLoad={[Function]}
        src="/_next/image?url=%2Fassets%2Fimages%2Ffacebook-logo.png&w=96&q=75"
        srcSet="/_next/image?url=%2Fassets%2Fimages%2Ffacebook-logo.png&w=48&q=75 1x, /_next/image?url=%2Fassets%2Fimages%2Ffacebook-logo.png&w=96&q=75 2x"
        style={
          {
            "color": "transparent",
          }
        }
        title="Login with Facebook"
        width={40}
      />
    </div>
    <div
      className="p-2 rounded shadow-md cursor-pointer"
    >
      <img
        alt="Github Logo"
        data-nimg="1"
        decoding="async"
        height={40}
        loading="lazy"
        onError={[Function]}
        onLoad={[Function]}
        src="/_next/image?url=%2Fassets%2Fimages%2Fgithub-logo.png&w=96&q=75"
        srcSet="/_next/image?url=%2Fassets%2Fimages%2Fgithub-logo.png&w=48&q=75 1x, /_next/image?url=%2Fassets%2Fimages%2Fgithub-logo.png&w=96&q=75 2x"
        style={
          {
            "color": "transparent",
          }
        }
        title="Login with GitHub"
        width={40}
      />
    </div>
  </section>
</section>
`;
